src/main/resources/application.yml

```diff
@@ -25,4 +18,51 @@ spring.datasource:
   password:
   max-active: 10
   max-idle: 1
-  min-idle: 1
\ No newline at end of file
+  min-idle: 1
+
+# Facebook Connection Details
+facebook:
+  client:
+    id: 140524976574963
```

Comment:
you okay with this stuff public?

Reply:
Similarly, the google client-ids probably shouldn't be in a public repo...

Reply:
@Jephuff could you please regenerate the facebook app credentials?